cpp/simulations/data_generation.sh

```diff
@@ -0,0 +1,42 @@
+#! /bin/bash
+# Run all scripts in memilio-epidata which are necessary to run the simulation files.
+
+# activate virtual_env 
+# The path to the virtual environment has to be adjusted individually
+# Otherwise the script can be called with -PATH_ENV Path argument, e.g.
+# sh data_generation.sh -PATH_ENV "HOME/Documents/virtualenv/bin/activate"
+path_virtual_env= "HOME/Documents/virtualenv/bin/activate" 
```

Comment:
please remove your particular path in docu and code

Reply:
It was not really my path. But i changed it to a more generic one.